Contrib/CCCS/AlgebraicLiteracy/IA_9.5/OpenStax_IA_9.5_203.pg

```diff
@@ -49,16 +49,26 @@ $a2 = -$c;
 $height = Compute("(-$b + sqrt($b**2 - 4*$a*$c))/(2*$a)");
 $base = Compute("2*$height + $m");
 
+$heightr = Round($height,1);
+$heightr = Compute ($heightr)->with(
+tolType => 'absolute',
+tolerance => 0.005,
+);
 
+$baser = Round($base,1);
+$baser = Compute ($baser)->with(
+tolType => 'absolute',
+tolerance => 0.005,
+);
 ###########################
 #  Main text
 
 BEGIN_PGML
-The area of a triangular flower bed in the park is [`[$A]`] square feet.  The base is [`[$m]`] feet longer than twice the height.  Find the base and height.  Round to the nearest tenth.
+The area of a triangular flower bed in the park is [`[$A]`] square feet.  The base is [`[$m]`] feet longer than twice the height.  Find the base and height.  Do not round until the end, then round to the nearest tenth of a foot.
 
 Height = [_______________]{$height}feet
 
-Base =  [_______________]{$base}feet
+Base =  [_______________]{$baser}feet
```

Comment:
Did you intend to change `$height` to `$heightr` here as well?

Reply:
Yes, yes we did. I will correct.